README.md

```diff
@@ -59,6 +59,46 @@ const config = {
 module.exports = { config };
 ```
 
+### API Configuration
+
+If you wish to use the Electron APIs then you will need to import the preload and main scripts. At the top of your preload:
```

Comment:
Was this the same for Spectron?

Reply:
Yeah, you need the handlers (main process) to hit the Electron APIs and the preload is required for the mapping over IPC...

Reply:
Gotcha, thanks for clarifying. My thought was that this might be a no-go for Electron devs to add this to their Electron application? Or is this concern irrelevant?

Reply:
It's a reasonable concern; when developing the fork I tried literally everything to avoid it, but it was made irrelevant when the Electron core devs tightened the security - there might be another way to do this at some point (like a testing API or something?) but right now I believe it's the only way...

Reply:
The problem is similar in `electron-redux`:

Old module, relying on insecure and out of date methods (now being rewritten):
https://github.com/klarna/electron-redux

New module, forked and updated to use modern methods:
https://github.com/aslilac/electron-redux